.drone.yml

```diff
@@ -14,7 +14,7 @@ steps:
 
 - name: build
   pull: default
-  image: rancher/dapper:1.11.2
+  image: rancher/dapper:v0.5.3
```

Comment:
Is this a downgrade or did the versioning scheme change?

Reply:
Good catch, let me update it. Thanks.

Reply:
No problem, looks like the dapper versioning scheme was changed.
@boknowswiki , @meldafrawi We should make sure to update to the the same version for all of our components.